lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -111,7 +111,7 @@ def define
         end
 
         next_node do
-          if %w(italy sweden).include?(calculator.ceremony_country)
+          if %w(italy sweden mozambique).include?(calculator.ceremony_country)
```

Comment:
The idea of a function is a good one. I think it needs a descriptive name though.